Shopify/shopify-cli
# Accept storefront renderer token from ENV variable

### WHY are these changes introduced?

Part of running CLI 2.0 from CLI 3.0.

This is necessary to support proxy requests to the theme dev server (i.e. hot reloading), which use a different token requiring the devtools scope.



### WHAT is this pull request doing?

Instead of just assuming you've logged in the CLI 2.0 way, accept a token passed from CLI 3.0 which specifies the storefront renderer scope.

We've already done something similar in [this PR](https://github.com/Shopify/shopify-cli/pull/2269).


### How to test your changes?


1. Check out [this branch](https://github.com/Shopify/cli/pull/206) of CLI3
2. Run `SHOPIFY_CLI_2_0_DIRECTORY=/path/to/Shopify/shopify-cli dev shopify theme dev --path=/path/to/your-theme --store=your-store.myshopify.com`
3. Everything should work as expected, including live reload!

### Update checklist
